dev-lang/clojure/clojure-1.10.3.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2
+
+SPEC_ALPHA_VER="0.2.194"
+CORE_SPECS_ALPHA_VER="0.2.56"
+
+TOOLS_NAMESPACE_VER="1.1.0"
+JAVA_CLASSPATH_VER="1.0.0"
+TOOLS_READER_VER="1.3.4"
+TEST_GENERATIVE_VER="1.0.0"
+DATA_GENERATORS_VER="1.0.0"
+TEST_CHECK_VER="1.1.0"
+
+DESCRIPTION="General-purpose programming language with an emphasis on functional programming"
+HOMEPAGE="https://clojure.org/"
+SRC_URI="
+	https://github.com/${PN}/${PN}/archive/${P}.tar.gz
+	https://github.com/clojure/spec.alpha/archive/spec.alpha-${SPEC_ALPHA_VER}.tar.gz
+	https://github.com/clojure/core.specs.alpha/archive/core.specs.alpha-${CORE_SPECS_ALPHA_VER}.tar.gz
+	test? (
+		https://github.com/clojure/tools.namespace/archive/refs/tags/tools.namespace-${TOOLS_NAMESPACE_VER}.tar.gz
+		https://github.com/clojure/java.classpath/archive/refs/tags/java.classpath-${JAVA_CLASSPATH_VER}.tar.gz
+		https://github.com/clojure/tools.reader/archive/refs/tags/tools.reader-${TOOLS_READER_VER}.tar.gz
+		https://github.com/clojure/test.generative/archive/refs/tags/test.generative-${TEST_GENERATIVE_VER}.tar.gz
+		https://github.com/clojure/data.generators/archive/refs/tags/data.generators-${DATA_GENERATORS_VER}.tar.gz
+		https://github.com/clojure/test.check/archive/refs/tags/test.check-${TEST_CHECK_VER}.tar.gz
```

Comment:
You might want to prepend `clojure-` or similar to the distfile names so it's clear what packages these files refer to

Reply:
We should consider it if we have distfiles collisions in the future